src/Umbraco.Web/PropertyEditors/RichTextPropertyEditor.cs

```diff
@@ -99,6 +112,11 @@ public override object FromEditor(Core.Models.Editors.ContentPropertyData editor
 
                 var editorValueWithMediaUrlsRemoved = TemplateUtilities.RemoveMediaUrlsFromTextString(editorValue.Value.ToString());
                 var parsed = MacroTagParser.FormatRichTextContentForPersistence(editorValueWithMediaUrlsRemoved);
+
+                var userId = Current.UmbracoContext.Security.CurrentUser.Id;
+
+                // TODO: In future task(get the parent folder from this config) to save the media into
+                parsed = TemplateUtilities.FindAndPersistPastedTempImages(parsed, -1, userId, _mediaService, _contentTypeBaseServiceProvider);
```

Comment:
```suggestion
                parsed = TemplateUtilities.FindAndPersistPastedTempImages(parsed, Constants.System.Root, userId, _mediaService, _contentTypeBaseServiceProvider);
```

Don't more introduce magic integers.